pkg/flowkit/account.go

```diff
@@ -121,3 +117,68 @@ func generateEmulatorServiceAccount(sigAlgo crypto.SignatureAlgorithm, hashAlgo
 		key:     NewHexAccountKeyFromPrivateKey(0, hashAlgo, privateKey),
 	}, nil
 }
+
+// Accounts is a collection of account.
+type Accounts []Account
+
+// Remove an account.
+func (a *Accounts) Remove(name string) error {
+	account := a.ByName(name)
+	if account == nil {
+		return fmt.Errorf("account named %s does not exist in configuration", name)
```

Comment:
This is just personal preference, but I feel Remove operations should be nilpotent.

Reply:
Do you mean idempotent or else please explain not sure what do you mean?

Reply:
Yes sorry, spelling mistake.

I meant that I should be able to call `Remove("x")` multiple times without any problems (errors). Like I said, just a personal preference. Feel free to ignore.

Reply:
yeah I kinda agree with you, idempotent functions are always better, will change